src/state/DummyStateServer.cpp

```diff
@@ -83,7 +83,7 @@ void DummyStateServer::start()
     });
 
     // Give it time to start
-    usleep(500 * 1000);
+    usleep(1000 * 1000);
```

Comment:
I found some of the state server tests were failing because of this short timeout.